.circleci/config.yml

```diff
@@ -106,14 +47,13 @@ jobs:
 
   e2e:
     docker:
-      - image: circleci/golang:1.15-node-browsers
+      - image: quay.io/influxdb/influxdb-circleci:latest
```

Comment:
Thought: we could use a `2.0` tag for the image instead of `latest` on this branch.